ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -2935,7 +2935,7 @@ private CadTemplate readMultiLeader()
 			}
 
 			//	BL Number of Block Labels 
-			int blockLabelCount = this._objectReader.ReadBitShort();
+			int blockLabelCount = this._objectReader.ReadBitLong();
```

Comment:
I guess that this line has been always been wrong but the value was always 0 so that's why we haven't had any problem, can you confirm it?